okta/resource_okta_policy_password.go

```diff
@@ -126,6 +126,14 @@ func resourcePolicyPassword() *schema.Resource {
 				Description: "If a user should be informed when their account is locked.",
 				Default:     false,
 			},
+			"lockout_notifications": {
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				MinItems:    1,
```

Comment:
I think you can drop this, I have no problems with someone specifying an empty set